ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -89,7 +89,7 @@ Item {
                 userName: ProfileUtils.displayName("", model.ensName, model.displayName, model.alias)
                 pubKey: model.isEnsVerified ? "" : Utils.getCompressedPk(model.pubKey)
                 isContact: model.isContact
-                isVerified: model.isVerified
+                isVerified: model.isVerified && !rootStore.isCurrentUser(model.pubKey)
```

Comment:
This is not the place to do it (as others pointed out); if we have another place that uses `StatusMemberListItem`, we'd have to replicate the logic quirk there too. Should be fixed on the NIM side

Reply:
Thanks! Fixed✅

Reply:
It's still here :grimacing:

Reply:
fixed (I forgot to add the file by mistake)

Reply:
You could have just removed the first commit; now you have 3 :) Don't forget to squash